src/metabase/legacy_mbql/util.cljc

```diff
@@ -2,6 +2,10 @@
   "Utilitiy functions for working with MBQL queries."
   (:refer-clojure :exclude [replace])
   (:require
+   #?@(:clj
+       [[metabase.legacy-mbql.jvm-util :as mbql.jvm-u]
+        [metabase.models.dispatch :as models.dispatch]
+        [metabase.util.i18n]])
```

Comment:
This hoisting is a bit unfortunate, but I guess its way to complex to match our old manual ordering - especially if that would require splitting or re-grouping to intersperse non-conditional namespaces 💀

Reply:
The only difference is that it's putting the conditional requires at the beginning instead of the end, there wasn't any sort of special manual ordering before, it just lived in a different place. We weren't consistent about putting them either at the beginning instead of the end previously, some namespaces did one while some did the other. So now it's enforcing one consistent style which is a good thingIMO

Reply:
Consistency for sure. I'm used to sorting them according to their contents, so sounds like there were 3 conventions before 🙀